src/main/java/org/testng/internal/ConfigurationMethod.java

```diff
@@ -455,5 +455,14 @@ public boolean isLastTimeOnly() {
     return result;
   }
 
+  public String[] getGroupFilters() {
+    String[] result = null;
```

Comment:
Can we please have this variable initialized to an empty array so that we can avoid a null check to be done ?